src/evergreen/api.py

```diff
@@ -361,6 +409,30 @@ def patches_by_project(self, project_id: str, params: Dict = None) -> Iterable[P
         patches = self._lazy_paginate_by_date(url, params)
         return (Patch(patch, self) for patch in patches)  # type: ignore[arg-type]
 
+    def configure_patch(
+        self,
+        patch_id: str,
+        variants: Optional[List[Dict[str, Union[str, List[str]]]]] = None,
+        description: Optional[str] = None,
+    ) -> None:
+        """
+        Update a patch.
+
+        :param patch_id: Id of the patch to update
+        :param variants: list of objects with keys "id" who's value is the variant ID, and key "tasks"
+        with value of a list of task names to configure for specified variant. See the documentation for more details
```

Comment:
Since this param string extends to multiple lines, our docs parser expects the lines after the first to be indented.